wpilib/wpilib/notifier.py

```diff
@@ -77,6 +92,8 @@ def _run(self) -> None:
                 if self._periodic:
                     self._expirationTime += self._period
                     self._updateAlarm()
+                else:
+                    self._updateAlarm(-1)
```

Comment:
You should add in the comment from upstream here explaining what this is for.

Reply:
I read that. And didn't comprehend it. And still don't. I'm kinda tired..